editor/src/clj/editor/prefs.clj

```diff
@@ -36,7 +37,8 @@
      (.toString out)))
 
 (defn- read-transit [s]
```

Comment:
Could add a type annotation here to get rid of the reflection warning. 
`^String`

Reply:
Ok, let my fix it